pyrpl/software_modules/lockbox/models/fabryperot.py

```diff
@@ -104,18 +83,19 @@ def calibrate(self, autosave=False):
         else:
             return None
 
-    def fit_with_voigt(self, curve, voltage):
+    def fit(self, curve, voltage):
         """ Fits curve to a Voigt profile. """
         x_for_fit = self.voltage_to_fit(voltage)
 
         peak = VoigtModel()
-        p_params = peak.guess(curve, x=x_for_fit)
+        p_params = peak.guess(curve, x=x_for_fit, negative=self.negative)
```

Comment:
Cool! I had no idea this was there. I see it in the [lmfit code](https://github.com/lmfit/lmfit-py/blob/82fd7742d184bc4f31b0604740f48d1b265c3c8b/lmfit/models.py#L450), but (at least not easily) in the lmfit docs. Here's hoping we don't need to touch  the maxima and minima. :crossed_fingers:

Reply:
up, you're right..